chapters/methodology.tex

```diff
@@ -29,3 +29,27 @@ \subsection{User Interface} \label{sec:user_interface}
 
 In terms of design, Android apps are generally expected to conform to Material Design, a set of guidelines defined by Google to help ensure both visual and practical consistency. As such, the user interface of the soHappy app is designed with Material Design in mind \cite{material_design}, incorporating principles such as animation or design based on the real world.
 
+\subsection{Model Training}
+
+As stated in the background section, soHappy uses a deep learning model to 
+provide information about a person smiling by utilizing TFLite.
+
+However, for TFLite to detect smiling, the model must be trained first. This
+model is not trained on the device, but will be trained in before and shipped
+with the android app.
+
+The soHappy smile detection model is on based a convolutional neural network 
+(CNN). CNNs are useful for image analysis purposes. A CNN utilizes multiple
+convolutional layers as well as pooling layers. A convolutional layer works by
+applying multiple different filters onto the input of the layer. An 
+convolutional layer also looks at regions of the applied filter layers instead 
+of individual pixels. This process is called convolution.
+Pooling describes the process of discarding unused information.
+Refer to \cite{8308186} for a more in-depth explanation on how CNNs work.
+
+For the soHappy smile detection model, eight convolutional layers are used. 
+Every two convolutional consecutive layers are followed by a pooling layer, 
+repeated four times. In the end, a model is flattened. This model

```

Comment:
Hm, I think it's more that the _results_ are flattened and a model is built _using_ those flattened results instead of the model itself being flattened.